index.html

```diff
@@ -289,6 +290,92 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div id="contact-me">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/mrgrnekq" method="POST">
+          <input
+            name="user_name"
+            type="text"
+            placeholder="Full name"
+            maxlength="30"
+          />
```

Comment:
Kindly make sure that a user cannot submit the form if the field is empty, as this will improve form validation prior to adding the information to your database

Reply:
Sure. Thanks